islandora_ocr.install

```diff
@@ -36,13 +36,16 @@ function islandora_ocr_enable() {
  */
 function islandora_ocr_cleanup_derivatives() {
   if (module_exists('islandora_book')) {
-    $book = variable_get('islandora_book_ingest_derivatives');
+    $book = variable_get('islandora_book_ingest_derivatives', array(
+      'pdf' => 'pdf',
+      'image' => 'image',
+      'ocr' => FALSE));
     $book['ocr'] = FALSE;
     variable_set('islandora_book_ingest_derivatives', $book);
   }
 
   if (module_exists('islandora_newspaper')) {
-    $newspaper = variable_get('islandora_newspaper_ingest_derivatives');
+    $newspaper = variable_get('islandora_newspaper_ingest_derivatives', array('ocr' => FALSE));
```

Comment:
@robyj Based on the above change and the default from the [newspaper solution pack's admin page](https://github.com/Islandora/islandora_solution_pack_newspaper/blob/7.x/includes/admin.form.inc#L36-L37) I think you should just do that same as above and set 
```
array (
  'pdf' => 'pdf',
  'image' => 'image',
  'ocr' => FALSE
)
```

Reply:
Done